src/subscription/cart/legal/SubscriptionLegal.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Hyperlink } from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+// import LocalizedPrice from '../../../payment/cart/LocalizedPrice';
+import messages from './messages';
+
+const SubscriptionLegal = ({
+  programTitle,
+  price,
+  intl,
+}) => {
+  const supportLink = (
+    <Hyperlink
+      destination="https://support.edx.org/hc/en-us/sections/115004173027-Receive-and-Share-edX-Certificates"
```

Comment:
Instead of hard coding the support URL here, can you get it from `getConfig()` instead and append the remainder? I believe it's `SUPPORT_URL`. It's best to get URLs from configs for open edX, but also in case they change in the future

Reply:
I will do it once we have actual URL from Michelle, However I have documented this in my follow up [ticket](https://2u-internal.atlassian.net/browse/PON-82).

Reply:
Great, thanks! We should remind her tomorrow about the URL.

Reply:
Resolving it for now already added as requirement in the next follow up ticket.